run_dir/design/ont_flowcells.html

```diff
@@ -134,13 +134,16 @@ <h1>
                     <td class="prep_kit">{{ ont_flowcells[onefc]["prep_kit"] }}</td>
                     <td class="barcoding_kit">{{ ont_flowcells[onefc]["barcoding_kit"] }}</td>
                     <td class="flow_cell_type">{{ ont_flowcells[onefc]["flow_cell_type"] }}</td>
-
                     <td class="flow_cell_qc text-center">
                         {% try %}
-                            {{ ont_flowcells[onefc]["lims"]["pore_count_history"][0]["num_pores"] }} 
+                            {{ [e for e in ont_flowcells[onefc]["pore_count_history"] if e["type"] == "qc"][0]["num_pores"] }}
```

Comment:
I'm leaning towards saying this is a bit too complicated logic inside a html-file. Especially with the nested try except. On the other hand, it's not that I have a ready suggestion for how it should be done instead.

Reply:
I initially had HTML in-line comments to document the cases, but this caused the sorting functionality of the JS DataTable to break. I can try to draft something a bit less messy.

Reply:
How is this? 4f4b0610387695d1a8caada47f5f0f60845ea56b

Reply:
That's too bad! Perhaps you could try jusing template comments `{# ... #}`. Ref: https://www.tornadoweb.org/en/stable/template.html#syntax-reference

Reply:
Much better! I only miss some specification of what is actually raising the exception

Reply:
Explicated error types in [e5c58f7](https://github.com/SciLifeLab/genomics-status/pull/771/commits/e5c58f77e4e98a549cc697389650e5bbd9766b96)

I think for the future it would be better to overhaul the back-end to reduce the complexity of the templating.